lib/tools.py

```diff
@@ -325,7 +325,7 @@ def sort_csv(filename):
     csv_reader_infile = csv.reader(input_file)
 
     # The first entry is the header line
-    infields = next(csv_reader_infile)
+    infields = ""
```

Comment:
Yes we do not need it, It's not compatible with the p3 version of csv engine. I will test it to document the error produced.